Decred Wallet/Features/More/More Features/Security Tools/Verify Message/VerifyMessageViewController.swift

```diff
@@ -191,6 +191,8 @@ class VerifyMessageViewController: UIViewController, FloatingPlaceholderTextView
         if capturedText.starts(with: "decred:") {
             capturedText = capturedText.replacingOccurrences(of: "decred:", with: "")
         }
+        self.addressText.textViewDidBeginEditing(self.addressText)
         self.addressText.text = capturedText
+        self.toggleValidateButtonState()
```

Comment:
A cleaner alternative would be to use `self.addressText.setText()` which simulates keyboard input.